src/scripts/changelog.zig

```diff
@@ -268,6 +272,7 @@ test ChangelogIterator {
         \\
     );
     try std.testing.expectEqualStrings(entry.text_body,
+        \\
```

Comment:
We probably don't want this leading newline?

Reply:
I guess it's there just to stress the parser